src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

```diff
@@ -128,7 +128,7 @@
             </trans-unit>
             <trans-unit id="35">
                 <source>This value should be a valid number.</source>
-                <target>This value should be a valid number.</target>
+                <target>This value should be a valid number or a valid datetime.</target>
```

Comment:
this is displayed in the final UI,isn't it?
in the UI context, it might be strange to display this "or" - in a field where only a number has meaning.
Is this valid concern? Can we do something about it?

Reply:
We could add a second invalid message and refactor the `RangeValidator` a little, so that at least you don't get the message `This value should be a valid number.`, when it's clear that you're validating datetimes, e.g. in the following scenario:

```php
$this->validator->validate('foo', new Range(['min' => '2020-01-01', 'max' => '2020-01-31']));
```

While we're at it, we could also add two more messages, so that when datetime exceeds limit, you would get an `earlier / later` info instead of `more / less`.

Reply:
I think adding dedicated messages is a better idea.

Reply:
As I understand, adding new message means, that I should switch to `master`. If so, I will close this PR and I will prepare a separate one.

Reply:
You can add a new message in 3.4 as it fixes a display bug.